src/write_resume_data.cpp

```diff
@@ -126,7 +126,7 @@ namespace libtorrent {
 		if (!atp.trackers.empty())
 		{
 			entry::list_type& tr_list = ret["trackers"].list();
-			tr_list.push_back(entry::list_type());
+			tr_list.emplace_back(entry::list_type());
```

Comment:
when using ``emplace_back()``, I don't think you need to pass a default-initialized object. By not passing any arguments, it should just default initialize the ``list_type``, right?